src/vm/tests/integrations.rs

```diff
@@ -254,6 +269,27 @@ fn integration_test_get_info() {
                         bhh, &contract_identifier, "(test-5)"),
                     Value::some(parent_val).unwrap());
 
+
+                let cur_tip = (chain_tip_info.burn_header_hash.clone(), chain_tip_info.anchored_header.block_hash());
+                // check that 1000 stx _was_ transfered to the contract principal
+                assert_eq!(
+                    chain_state.with_read_only_clarity_tx(&cur_tip.0, &cur_tip.1, |conn| {
+                        conn.with_clarity_db_readonly(|db| {
+                            db.get_account_stx_balance(&contract_identifier.clone().into())
+                        })
+                    }),
+                    1000);
```

Comment:
Now that the contract principal is materialized and credited, can we test that:

* You can still deploy a smart contract to that contract address, even though it's now materialized and has a balance?
* You continue to only be able to deploy code to a smart contract address at most once, even if it was first materialized without code?
* You can send STX to an existing smart contract, which the smart contract can then spend with `stx-transfer`?